spa/src/components/content/pages/AddPageModal.js

```
@@ -54,7 +54,11 @@ function AddPageModal({ isOpen, closeModal, pagesByRevenueProgram }) {
     (e) => {
       setLoading(false);
       if (e.response?.data) {
-        setErrors(e.response.data);
+        if (revenuePrograms?.length === 1) {
```

Is this the bug fix? e.g. before it was failing silently?

---

Yes, before it wouldn't show the error `you have reached the page limit of 1` (or something like this)

---

added tests for this